site/content/sponsors/informedk12.md

```diff
@@ -2,5 +2,5 @@
 title: InformedK12
 description: Easy-to-use workflow automation solution that helps school district administrators digitize forms, automate processes and track approvals.
 link: https://www.informedk12.com/careers
-image: https://www.informedk12.com/hubfs/Logo%20%5BJune-17%5D/mark-rgb-isolated.png?t=1536235567849
+image: https://user-images.githubusercontent.com/4284691/48388066-f0972200-e6ac-11e8-94a9-fb27a8c29512.png
```

Comment:
@YourMobileGeek Can we get these added in locally instead of by URL? 🙇‍♀️ You can see examples of this if you search 'assets/img/'

Reply:
That way, they get optimized automagically 😄

Reply:
Yup I'll go ahead and import them that way, I was placing them in the static folder and was wondering why it wasn't showing up. 😅